### PR TITLE
remove django-bootstrap3 dependencie

### PR DESCRIPTION
**Pull request by drgarcia1986**
I still can't test, because I'm lost in allauth flow

### CODE REVIEW

File: houserules/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		{% block head %}
+			<meta charset=utf-8 />
+			<title>
+                {% block title %}Houserules{% endblock title %}
+			</title>
+			{% block style %}
+				<link href="https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css" rel="stylesheet" media="screen">
```
Comment (renanivo) [inline houserules/templates/base.html]: `v4-dev`? Is it stable enough?

Comment (drgarcia1986) [inline houserules/templates/base.html]: IDK :joy: but I'm change to 3.3.6

**Comment by drgarcia1986** [comment]
I'm stop at moment and rethink this PR. 
We will use forms and others complex HTML components ? 
If yes, maybe django-bootstrap3 can help us.

**Comment by renanivo** [comment]
I think we will, but right now nothing is permanent. Let's just try to create something minimally usable and please the eyes later.

**Comment by drgarcia1986** [comment]
LGTM :+1: 
Let's do this merge ?

**Pull request merged**